pulpcore/cli/common/generic.py

```diff
@@ -655,10 +655,22 @@ def _type_callback(ctx: click.Context, param: click.Parameter, value: Optional[s
     click.option(
         "--download-concurrency", type=int, help=_("total number of simultaneous connections")
     ),
-    click.option("--password"),
+    click.option(
+        "--password",
+        help=_(
+            "The password to authenticate to the remote. "
+            "Extra leading and trailing whitespace characters are not being trimmed."
```

Comment:
It's a bit strange to talk about something that is not being done, unless it were usually expected.
Maybe "The password including surrounding whitespace to authenticate with ..."

Reply:
The `surrounding whitespace` may suggest adding whitespace characters to the original password. I would rather leave it unchanged. Or, I can altogether remove the help message because the trimming is not expected by default ...

Reply:
The added complexity here is that you usually issue those commands in bash, and that _will_, if not quoted correctly, trim your passwords again. Also tabs and linebreaks in passwords are really problematic for any UI.
How about: "The password to authenticate to the remote (can contain leading and trailing spaces)."?